### PR TITLE
Fixed inconsistency with using hashedPassword

### PR DESCRIPTION
**Pull request by edamamez**
string password is now hashedPassword wherever it is used.

### CODE REVIEW

File: src/main/java/codeu/model/data/User.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
   public User(UUID id, String name, String password, Instant creation) {
```
Comment (kevmo314) [inline src/main/java/codeu/model/data/User.java]: Can you rename the parameter in the constructor too?

File: src/main/java/codeu/model/data/User.java [hunk 2]
```diff
@@ -52,14 +52,14 @@ public String getName() {
 
   /** Returns the password of this User. */
   public String getPassword() {
```
Comment (kevmo314) [inline src/main/java/codeu/model/data/User.java]: Also should this be refactored to be called `getHashedPassword()`?

Comment (ezhou20) [reply src/main/java/codeu/model/data/User.java]: Sure!

Comment (ezhou20) [reply src/main/java/codeu/model/data/User.java]: Oh well if we change it we will have to fix everywhere it is used...

Comment (ezhou20) [reply src/main/java/codeu/model/data/User.java]: Oh nevermind I understand, 'refactor' haha. I'm just using atom so I'm not sure if I can but I'll try

**Pull request merged**